examples/FHIR-to-tables/FHIR-to-tables.ipynb

```diff
@@ -0,0 +1,123 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FHIR to Tables\n",
+    "\n",
+    "This notebook provides a tutorial on how to use functionality from the PHDI_building_blocks library to achieve the following:\n",
+    "1. Read a user-defined schema config file that specifies one or more tables.\n",
+    "2. Query a FHIR server to extract the data specified in the schema.\n",
+    "3. Transform the data into a structured format.\n",
+    "4. Load the data into one of the supported persistent tabular file formats.\n",
+    "5. Print a summary of the tables that were created.\n",
+    "\n",
+    "### Getting Started\n",
+    "\n",
+    "To make it easier to get started we have included two additional files in this direcotry:\n",
+    "1. A simple example schema config file called `example_schema.yaml`.\n",
+    "2. A `requirements.txt` file that can be used to install the `PHDI_building_blocks` library in your Python environment by running `pip install -r requirements.txt`.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Import Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
```

Comment:
It seems odd to me that we are committing the execution count as well as notebook metadata down below.  It looks like there are solutions to clean this information when committing.  Is that something we should look in to?

Reply:
This is a really great point. Just did a little reading on nb-clean, going to look at setting up a GitHub action to ensure notebooks are clean before allowing a merge.

Reply:
Added an additional job in the Test workflow to run [this](https://github.com/ResearchSoftwareActions/EnsureCleanNotebooksAction) git hub action that ensures notebooks have clean metadata. I have confirmed that `nb-clean` is capable of cleaning notebook metadata and that its use results in a notebook that was previously failing the new test to pass it.